.travis.yml

```diff
@@ -3,7 +3,6 @@ dist: trusty
 language: node_js
 node_js:
   - 8
-  - 9
```

Comment:
Removed node 9 from test versions, to speed up builds. Node 9 is not an LTS version, and node 10 is not supported by original CRA anyway. After migrating to CRA v2 node 10 will be added.